uber/templates/jobs/staffers.html

```diff
@@ -70,7 +70,7 @@
 <div style="margin-left:10px">
     [D]: <i>Staffers marked with a "D" are department heads.</i> <br/>
     [M]: <i>Staffers marked with a "M" are assigned to multiple locations</i> <br/>
-    [T]: <i>Staffers marked with a "T" are trusted</i> <br/>
+    [T]: <i>Staffers marked with a "T" are trusted in at least one department</i> <br/>
     [L]: <i>Staffers marked with a "L" are working load-in and/or load-out</i>
 </div>
 
```

Comment:
correct, yea, I think I have about 3 of these left.  I will definitely ping this PR with a 'I THINK I AM ACTUALLY DONE NOW, PLEASE FINAL REVIEW', before merging.

plus I still have to write the migration